src/containers/shared/components/Transaction/AMM/AMMCreate/test/AMMCreate.test.tsx

```diff
@@ -0,0 +1,21 @@
+import { Simple } from '../../AMMSharedSimple'
+import { createSimpleWrapperFactory } from '../../../test'
+import createMock from './mock_data/amm_create.json'
+
+describe('AMM Create Tests', () => {
+  const createWrapper = createSimpleWrapperFactory(Simple)
+
+  it('renders from transaction', () => {
+    const wrapper = createWrapper(createMock)
+    expect(wrapper.find('[label="asset1"] .value')).toHaveText(
```

Comment:
There are helper methods for this expectSimpleRow*

Reply:
done - ill make this change to the other test files as well